yt_dlp/extractor/rcti.py

```diff
@@ -0,0 +1,242 @@
+            'channel_id': video_json.get('tv_id'),
+            'formats': formats,
+            'thumbnails': thumbnails
+        }
+
+
+class RCTIPlusSeriesIE(RCTIPlusBaseIE):
+    _VALID_URL = r'https://www\.rctiplus\.com/programs/(?P<id>\d+)/(?P<display_id>[^/?#&]+)(?:\W)*$'
+    _TESTS = [{
+        'url': 'https://www.rctiplus.com/programs/540/upin-ipin',
+        'playlist_mincount': 417,
+        'info_dict': {
+            'id': '540',
+            'title': 'Upin & Ipin',
+            'description': 'md5:22cc912381f389664416844e1ec4f86b',
+        },
+    }, {
+        'url': 'https://www.rctiplus.com/programs/540/upin-ipin/#',
+        'only_matching': True,
+    }]
+    _AGE_RATINGS = {  # Based off https://id.wikipedia.org/wiki/Sistem_rating_konten_televisi with additional ratings
+        'S-SU': 2,
+        'SU': 2,
+        'P': 2,
+        'A': 7,
+        'R': 13,
+        'R-R/1': 17,  # Labelled as 17+ despite being R
+        'D': 18,
+    }
+
+    def _entries(self, url, display_id=None, note='Downloading entries JSON', metadata={}):
+        total_pages = 0
+        try:
+            total_pages = self._call_api(
+                '%s&length=20&page=0' % url,
+                display_id, note)[1]['pagination']['total_page']
+        except ExtractorError as e:
+            if 'not found' in str(e):
+                return []
+            raise e
+        if total_pages <= 0:
+            return []
+
+        for page_num in range(1, total_pages + 1):
+            episode_list = self._call_api(
+                '%s&length=20&page=%s' % (url, page_num),
+                display_id, '%s page %s' % (note, page_num))[0] or []
+
+            for video_json in episode_list:
+                link = video_json['share_link']
+                url_res = self.url_result(link, 'RCTIPlus', video_json.get('product_id'), video_json.get('title'))
+                url_res.update(metadata)
+                yield url_res
+
+    def _real_extract(self, url):
+        series_id, display_id = re.match(self._VALID_URL, url).groups()
+
+        series_meta, meta_paths = self._call_api(
+            'https://api.rctiplus.com/api/v1/program/%s/detail' % series_id, display_id, 'Downloading series metadata')
+        metadata = {
+            'age_limit': try_get(series_meta, lambda x: self._AGE_RATINGS[x['age_restriction'][0]['code']])
+        }
+
+        cast = []
+        for star in series_meta.get('starring', []):
+            cast.append(strip_or_none(star.get('name')))
+        for star in series_meta.get('creator', []):
+            cast.append(strip_or_none(star.get('name')))
+        for star in series_meta.get('writer', []):
+            cast.append(strip_or_none(star.get('name')))
+        metadata['cast'] = cast
+
+        tags = []
+        for tag in series_meta.get('tag', []):
+            tags.append(strip_or_none(tag.get('name')))
+        metadata['tag'] = tags
+
+        entries = ()
```

Comment:
```suggestion
        entries = []
```
Don't make tuple if you are going to `+` anyway